plugins/hls-semantic-tokens-plugin/hls-semantic-tokens-plugin.cabal

```diff
@@ -52,12 +52,15 @@ library
     , array
     , deepseq
     , hls-graph == 2.5.0.0
+    , data-default
+    , rank2classes
```

Comment:
we don't currently depend on this so it would be mildly better to avoid it if we can. It still looks actively developed, but you never know, and often small stuff like this lags with GHC version upgrades etc.

Reply:
It just provide a rank2 liftA2. We can customize one of these.

Reply:
removed to in favor of template haskell